src/Events/SettingUpdated.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Why is this added here? We don't have it elsewhere in the project.  It may be a good idea, I don't know, but I'm a bigger fan of consistency.  Please remove it from this and other files.

Reply:
I will, it's one of the things from my php Cs fixer.

Once one of you approve that the actual code is alright I'll give it a once over and remove any styleCI issues as well as these. I think I also wrote it in the comment above.